js/ui/form/ui.form.layout_manager.js

```diff
@@ -708,14 +708,15 @@ const LayoutManager = Widget.inherit({
     },
 
     _createComponent: function($editor, type, editorOptions) {
-        const that = this;
         const readOnlyState = this.option('readOnly');
-        const instance = that.callBase($editor, type, editorOptions);
-
-        readOnlyState && instance.option('readOnly', readOnlyState);
+        const hasEditorReadOnly = isDefined(editorOptions.readOnly);
```

Comment:
I think we should we keep `editorOptions.readOnly` value even if it's specified as an `undefined` or `null` while it casts to `false` in standalone editors.
I guess it should be smth like this
```suggestion
        const hasEditorReadOnly = Object.hasOwn(editorOptions, 'readOnly');
```

Reply:
If `editorOptions.readOnly` is undefined, it is not will work as null or false with this changes. And I think we should not to make behavior with `undefined` equal to null and false values.

Reply:
okay let's leave it now